wiki-refresh

```diff
@@ -55,44 +46,15 @@ if [ "${DATA_DIR}" = "" ]; then
   exit 1
 fi
 
-# Make sure we have csvkit by checking version on csvgrep.
-# If we don't have it, one way to get it would be:
-#
-#   $ git clone https://github.com/wireservice/csvkit.git
-#   $ cd csvkit
-#   $ sudo python3 setup.py install --prefix=/usr/local
-check_for_bin csvgrep
-csvgrep --version 2&>/dev/null || { echo "csvkit needs to be a recent version. Please upgrade to version 1.0.2+"; exit 1; }
+# Make sure we have python3
+python3 -V > /dev/null 2>&1 || { echo "python3 required, aborting" ; exit 1; }
 
-# Make sure we have csv2wiki.
-CSV2WIKI="${SOURCES_ROOT}/csv2wiki/csv2wiki"
-if [ ! -x ${CSV2WIKI} ]; then
-  CSV2WIKI=${OTSDIR}/clients/macfound/eval-system/csv2wiki/csv2wiki
-fi
-if [ ! -x ${CSV2WIKI} ]; then
-  CSV2WIKI=${OTSDIR}/clients/macfound/eval-system/r/csv2wiki/csv2wiki
-fi
-if [ ! -x ${CSV2WIKI} ]; then
-  CSV2WIKI=${OTSDIR}/csv2wiki/csv2wiki
-fi
-if [ ! -x ${CSV2WIKI} ]; then
-  CSV2WIKI=${OTSDIR}/r/csv2wiki/csv2wiki
-fi
-if [ ! -x ${CSV2WIKI} ]; then
-  # Okay, let's get medieval on the situation.
-  (cd "${SOURCES_ROOT}"; \
-   git clone https://github.com/OpenTechStrategies/csv2wiki.git)
-  CSV2WIKI="${SOURCES_ROOT}/csv2wiki/csv2wiki"
-fi
-if [ ! -x ${CSV2WIKI} ]; then
-  echo "ERROR: Unable to find or clone 'csv2wiki'."
-  exit 1
-fi
-# Test that csv2wiki actually works.
-if ${CSV2WIKI} --help > /dev/null; then
+python3 -m csv2wiki --version >/dev/null 2>&1 || { echo "csv2wiki not installed, make sure 'python3 -m csv2wiki' works required, aborting" ; exit 1; }
```

Comment:
Does this code now assume that it's running from the top of the `csv2wiki` tree?